packages/js/integrate-plugin/src/get-plugin-config.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { existsSync, readFileSync, promises } from 'fs';
+import { join } from 'path';
+
+/**
+ * Internal dependencies
+ */
+import { info } from './log';
+
+const { writeFile } = promises;
+function getUniqueItems( arr: string[] ) {
```

Comment:
I see that there is some unused code here. I'm assuming it will be used in the future (if so, that's fine).